runatlantis.io/docs/using-atlantis.md

```diff
@@ -167,6 +167,49 @@ atlantis import -d dir 'aws_instance.example["foo"]' i-1234567890abcdef0 -- -var
 ```
 If a flag is needed to be always appended, see [Custom Workflow Use Cases](custom-workflows.html#adding-extra-arguments-to-terraform-commands).
 
+---
+## atlantis state rm
```

Comment:
Do you plan to implement `state mv` too ?

Reply:
Hmm i guess one more reason to upgrade. Im working with a client now that uses old versions of terraform and hasnt had the cycles to upgrade. I imagine others are also in the same boat.